backend/module/chaos/serverexperimentation/server_experimentation_integration_test.go

```diff
@@ -0,0 +1,233 @@
+// +build integration_only
```

Comment:
These tests has to be moved to `serverexperimentation` file due to some circular dependency issue that occurred for when these tests stayed in`expermentation` (this is due to the fact that `xds` pieces were moved from `serverexperimentation` to `experimentation` package